pom.xml

```diff
@@ -68,6 +68,13 @@
 
     <aerius-tools.version>1.1.1</aerius-tools.version>
     <spotless.version>2.5.0</spotless.version>
+
+    <sonar.language>java</sonar.language>
+    <sonar.maven.plugin.version>3.7.0.1746</sonar.maven.plugin.version>
```

Comment:
Can't we use the latest version of the plugin just like in the AERIUS-II repository?